doc/develop/flash_debug/host-tools.rst

```diff
@@ -270,6 +270,10 @@ LinkServer west runner   ``--probe`` option to pass the probe index.
 
    west flash --runner=linkserver --override /device/memory/5/flash-driver=MIMXRT500_SFDP_MXIC_OSPI_S.cfx
 
+4. LinkServer does not install an implicit breakpoint at the reset handler. If
+   a user would like to single step from the start of their application, the
+   will need to add a breakpoint at ``main`` or the reset handler manually.
```

Comment:
Please consider addressing the user as "you" to be consistent with the rest of the page. You may also get rid of the typo ("the" instead of "they") in the process:) thanks!

Reply:
Thanks for catching both of these issues- updated the PR to fix them